doc-content/enterprise-only/installation/clustering-smart-router-install-proc.adoc

```diff
@@ -7,6 +7,15 @@ This section describes how to install Smart Router and configure it for a Red Ha
 
 .Prerequisites
 * {KIE_SERVER} is installed on each node of a {EAP} {EAP_VERSION} cluster.
++
+[NOTE]
+====
+Smart Router does not require {KIE_SERVER} to be configured as unmanaged.
```

Comment:
Suggested rewrite:
It is not required to configure an unmanaged KIE_Server to a Smart Router. //To avoid the passive form.

Reply:
You're right to try and avoid the passive form. The state for KIE_SERVER is a boolean (either managed or unmanaged) and it is not a requirement for Smart Router. Maybe:

"You do not need to configure KIE_SERVER as unmanaged for Smart Router."

Reply:
yeah looks good